content/news/20160629-closing-the-cfp.md

```diff
@@ -0,0 +1,11 @@
+type: news-article
+category: news
+title: Closing the CFP
+url: /news/20160629-closing-the-cfp/
+datestamp: 20160629
+humandate: 29th June 2016
+---
+
+We have now closed the  [Call for Proposals](/cfp/) for talks, and we'll be busy over the next few weeks putting together a programme that we hope will be interesting and enjoying.
```

Comment:
I'd strengthen the wording a little here and remove the 'hope' element:

"...and we'll be busy over the next few weeks putting together a an interesting and exciting programme."